lib/connect/ws.js

```diff
@@ -2,14 +2,36 @@
 
 var websocket = require('websocket-stream')
 var _URL = require('url')
-var wssProperties = [
-  'rejectUnauthorized',
-  'ca',
-  'cert',
-  'key',
-  'pfx',
-  'passphrase'
-]
+var mqttProperties = [
+'keepalive',
+'reschedulePings',
+'clientId',
+'protocolId',
+'protocolVersion',
+'clean',
+'reconnectPeriod',
+'connectTimeout',
+'username',
+'password',
+'incomingStore',
+'outgoingStore',
+'queueQoSZero',
+'will',
+];
+
+var urlKeys = [
+'href',
+'slashes',
+'path',
+'protocol',
+'auth',
+'hostname',
+'port',
+'host',
+'pathname',
+'search',
+'query',
+'hash' ];
```

Comment:
there are two ways to i guess: 
-set a list of options we want to support 

-or support everything and remove the overhead